test/controllers/questionnaire_controller_test.exs

```diff
@@ -117,15 +118,15 @@ defmodule Ask.QuestionnaireControllerTest do
 
     test "updates project updated_at when questionnaire is created", %{conn: conn, user: user}  do
       datetime = Ecto.DateTime.cast!("2000-01-01 00:00:00")
-      project = insert(:project, user: user, updated_at: datetime)
+      project = create_project_for_user(user)
```

Comment:
Was this `updated_at: datetime` used in the test? It's strange that this doesn't make travis fail... maybe the test wasn't correct after all.